src/main/scala/firrtl/passes/Passes.scala

```diff
@@ -86,9 +86,12 @@ object PassUtils extends LazyLogging {
        val p = passes.head
        val name = p.name
        logger.debug(s"Starting ${name}")
+       println(s"Starting ${name}")
        val x = p.run(c)
        logger.debug(x.serialize)
        logger.debug(s"Finished ${name}")
+       println(x.serialize)
```

Comment:
Please remove these printlns, I'm assuming they were just for debugging anyway.

They make the Travis log impossible to read (and since it's failing, we need to figure out why). Also, those logger.debugs will print what you need, you can locally change how much is printed by the logger in src/main/resource/logback.xml. Simply change root level to be "debug" instead of "warn"